### ISSUE TITLE
IPOPS detailed_ip_type: wrong for PRIVATE IP

### ISSUE THREAD

**Issue by homer5000** [kamailio/kamailio#1906]
### Description

I use the detailed_ipv4_type func from the ipops module for determine the source IP of the request:
```
# check src ip
if (detailed_ip_type("$si", "$var(src_t)")) {
    xlog("L_INFO","Sourece IP address ($si) is of detailed type: $var(src_t)\n");
    ...
```
When $si = 172.22.32.140 detailed_ip_type return PUBLIC, but it is PRIVATE.

#### Log Messages

```
INFO: <script>: Sourece IP address (172.22.32.140) is of detailed type: PUBLIC
```

### Additional Information

Maybe the reason is here:
modules / ipops / detailed_ip_type.c
```
(Lucian Balaceanu         2016-09-01 18:39:55 +0300 48)
     { 0xac100000,  "PRIVATE",    0xfffe0000 },  // 172.16/12
```
Maybe fffe should be fff0 - it's /15 now.. so 172.16 and 172.17 only match.

**Comment by miconda** [comment]
Can you make a pull request with the proposed changes? Then it can be reviewed and merged if all ok.

**Comment by sergey-safarov** [comment]
According [RFC1918](https://tools.ietf.org/html/rfc1918#section-3) need use 172.16/12
I will create PR

**Comment by henningw** [comment]
Thank you, fixed in git master, 5.2 and 5.1 branch.
